benchmarks/src/main/java/okhttp3/benchmarks/ApacheHttpClient.java

```diff
@@ -42,9 +41,9 @@
     super.prepare(benchmark);
     ClientConnectionManager connectionManager = new PoolingClientConnectionManager();
     if (benchmark.tls) {
-      SSLContext sslContext = SslContextBuilder.localhost();
+      SslContextBuilder sslContextBuilder = SslContextBuilder.localhost();
```

Comment:
nit: I’d prefer to just immediately call `build()` and not increase the visibility of the mutable `builder` thing